test/queryFilters/interpretGreaterOrEqual.test.ts

```diff
@@ -379,7 +382,8 @@ describe('interpretGreaterOrEqual', () => {
     ) as UnknownFilter;
 
     expect(filter).toEqual({
-      type: 'unknown'
+      type: 'unknown',
+      withError: { message: 'Function referenced is not "CalendarAgeInYearsAt"' }
     });
```

Comment:
Can someone double check these last 3 error messages to ensure that they are appropriate for the given situation.